english/candyland/swim/horizon/milkshake_everywhere.md

```diff
@@ -0,0 +1,5 @@
+You doggy-paddle in a circle to get a better look at your surroundings.  Milkshake, milkshake, everywhere!  Milkshake as far as the eye can see.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.